pkg/http/api_test.go

```diff
@@ -696,7 +696,7 @@ func TestBulkPubSubEndpoints(t *testing.T) {
 	})
 
 	t.Run("Bulk Publish with topic name '/' - 204", func(t *testing.T) {
-		apiPath := fmt.Sprintf("%s/publish/bulk/pubsubname//", apiVersionV1alpha1)
+		apiPath := fmt.Sprintf("%s/publish/bulk/pubsubname/%%2F", apiVersionV1alpha1)
```

Comment:
Yes, because in `fmt.Sprintf` the `%` is otherwise interpreted as a formatting one (like in `%f`). So to have an actual `%`, it needs to be doubled as `%%`